spec/ParseObject.spec.js

```diff
@@ -12,6 +12,7 @@
 // single-instance mode and we don't want these tests to run in
 // single-instance mode.
 
+const Parse = require('parse/node');
```

Comment:
This should not be necessary. There surely are existing tests with custom obj ID?